### ISSUE TITLE
Bug with DiscordSRV

### ISSUE THREAD

**Issue by Ter224** [RoinujNosde/SimpleClans#429]
Hello! I can't configure integration with DiscordSRV.
SimpleClans creates channels on my Discord server. But I ran into such problems:
- Roles are not given to clan members
- Does not add participants to the channel
- When sending a message to the clan chat, an error appears in the console. I have attached a screenshot

I tried to add participants to the channel, manually assign a role by the name of the clan - the result is the same.

**Software:**
 - Server: Purpur 1.16.5-1171
 - SimpleClans - latest build, 421
 - DiscordSRV 1.27.0 version
 
 
![Screenshot_2](https://github.com/RoinujNosde/SimpleClans/assets/94737619/3d89f520-5e3d-410a-9e37-def3cee956b2)

**Comment by RoinujNosde** [comment]
Are you using the bot in more than one discord server?
If not, check your config, as some ID is probably invalid.

@Tomut0

**Comment by Ter224** [comment]
> more than one discord server

Yes, it is. Later, I will create a separate bot just for SimpleClans and write about the results. Thanks.
